vignettes/git_usage.Rmd

```diff
@@ -43,9 +43,7 @@ Each feature branch must be related to an issue. We encourage new developers to
 
 ### Naming Branches
 
-The name of the branch must be prefixed with the issue number, followed by a short but meaningful description and the `@<target_branch>` suffix. The latter would be `@devel` in most cases. As an example, given an issue #94 "Program function to derive `LSTALVDT`", the branch name would be `94_derive_var_lstalvdt@devel`.
-
-The `@<target_branch>` suffix is used in our CI/CD pipelines, e.g. when running `R CMD check`. It ensures that `{admiral}`'s dependencies such as `{admiral.test}` and `{admiraldev}` are installed from the specified target branch. So when the target branch is set to `@devel` the dependencies will be installed from those package's respective `devel` branches rather than installing the latest released version. This ensures that we test the development version of `{admiral}` against the development versions of its  dependencies. That way all packages are kept in sync.
+The name of the branch must be prefixed with the issue number, followed by a short but meaningful description and the `@<target_branch>` suffix, if it requires building off a branch of an upstream repo. The `@<target_branch>` suffix is used in our CI/CD pipelines, e.g. when running `R CMD check`. It ensures that `{admiral}`'s dependencies such as `{pharmaversesdtm}` and `{admiraldev}` are installed from the specified target branch. So when the target branch is set to `@main` the dependencies will be installed from those package's respective `main` branches rather than installing the latest development version. This ensures that we test `{admiral}` against the corresponding versions of its dependencies. That way all packages are kept in sync.
```

Comment:
@pharmaverse/admiral When we changed the default branch to `@devel` did it make any impact on how we should name our branches? Or should this actually remain unchanged I think i misinterpreted what staged dependencies actually does, if anyone can chime in

Reply:
The fall-back branch of staged dependencies is `main` regardless of the default branch in github. I.e., if `@devel` is not added to the branch name, the CI/CD pipelines use the `main` branch of the upstream dependencies.